src/test/java/com/example/raisetechcoursetask10/service/SkiresortServiceImplTest.java

```diff
@@ -113,4 +116,24 @@ class SkiresortServiceImplTest {
         verify(skiresortMapper,times(1)).findById(1);
         verify(skiresortMapper, times(1)).deleteSkiresort(1);
     }
+
+    @Test
+    public void 新規のスキー場情報を登録できること() {
+        // skiresortCreateForm変数をインスタンス化して、それぞれの属性に値を設定
+        SkiresortCreateForm skiresortCreateForm = new SkiresortCreateForm(1, "CoronetPeak", "NZ", "初中級者の時に行ったので初めてのTバーに撃沈。岩だらけの広い氷山には木が１本もなくて、ボードと心が折れる人続出。上手くなってから行くべきゲレンデ");
+        Skiresort skiresort = new Skiresort(0, "CoronetPeak", "NZ", "初中級者の時に行ったので初めてのTバーに撃沈。岩だらけの広い氷山には木が１本もなくて、ボードと心が折れる人続出。上手くなってから行くべきゲレンデ");
+
+        // スタブ化　insertSkiresortの戻り値はvoidのためdoNothingを使用する
+        doNothing().when(skiresortMapper).insertSkiresort(skiresort);
+        // 登録テストを実行　createSkiresortメソッドを呼び出してskiresortを渡す
+        // 2回呼び出してしまうため、コメントアウト
+//        skiresortServiceImpl.createSkiresort(skiresortCreateForm);
+
+        // actualにはSkiresortのオブジェクト（skiresort）を代入する
+        Skiresort actual = skiresortServiceImpl.createSkiresort(skiresortCreateForm);
+        // skiresortServiceImple.createSkiresortの戻り値であるSkiresortオブジェクトの値が期待通りであるかを検証する
+        assertThat(actual).isEqualTo(skiresort);
+        // verifyでskiresortMapper.insertSkiresortが1回呼ばれて引数にSkiresortが渡されていることを検証する
+        verify(skiresortMapper,times(1)).insertSkiresort(skiresort);
```

Comment:
actions on save入れていない？
たぶん2つの端末で作業していると思うのですが、両方の端末のIntelliJにactions on saveの設定いれていますか？
```suggestion
        verify(skiresortMapper, times(1)).insertSkiresort(skiresort);
```

Reply:
操作している端末はmac1つです。
actions on save確認しましたが、どこにもチェックは入っていません。
毎回設定がリセットされると伺っていたのに失念していました。
チェック入れておきます。